src/app/services/crypt.js

```diff
@@ -20,6 +20,18 @@ module.exports = (Model, App) => {
     }
   }
 
+  function decryptNameWithNullFolderId(content) {
+    try {
+      const decryptedText = AesUtil.decrypt(content, null);
+
+      return decryptedText;
+    } catch (error) {
+      log.error(`(decryptNameWithNullFolderId): ${error}`);
+
+      return null;
+    }
+  }
```

Comment:
Had problems with `decryptName` as it uses probabilistic approach if no salt provided, so I had to get directly the Aes implementation.